pkg/apis/openebs.io/v1alpha1/cstor_completed_backup.go

```diff
@@ -23,22 +23,22 @@ import (
 // +genclient
 // +genclient:noStatus
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
-// +resource:path=backupcstorlast
+// +resource:path=cstorcompletedbackup
 
-// BackupCStorLast describes a cstor last-backup resource created as custom resource
-type BackupCStorLast struct {
+// CStorCompletedBackup describes a cstor completed-backup resource created as custom resource
+type CStorCompletedBackup struct {
```

Comment:
Can there be multiple completed backups?
Does it make sense to have `CStorLastCompletedBackup`

Reply:
Is this only going to store the recently completed backup? Then CStorRecentlyCompletedBackup makes sense.